ga_core/settings.py

```diff
@@ -209,49 +209,55 @@
 
 
 # Settings for logging
-log_dir = os.path.join(BASE_DIR, "logs")
+if not os.path.exists(f'/home/ga/logs/ga-error.log'):
+    with open(f'/home/ga/logs/ga-error.log', 'w'): pass
+if not os.path.exists(f'/home/ga/logs/ga-debug.log'):
+    with open(f'/home/ga/logs/ga-debug.log', 'w'): pass
+
 LOGGING = {
     'version': 1,
     'disable_existing_loggers': False,
     'formatters': {
-        'timestamp': {
```

Comment:
what do the logs look like now?

Reply:
Current Debug Log
![Screenshot from 2022-06-27 13-19-19](https://user-images.githubusercontent.com/60819172/175939623-a7f7cb98-e95d-47a4-834c-592f61505118.png)
Current Error Log
![Screenshot from 2022-06-27 13-19-20](https://user-images.githubusercontent.com/60819172/175939704-dcea0ced-ba0c-420d-8934-c14810e7d240.png)

I did not include an INFO log because it seems to overlap too much with DEBUG and ERROR. ERROR include just ERROR and above. DEBUG include everything.